README.md

```diff
@@ -129,7 +129,7 @@ Sometimes it's just easier to start over, cover my own needs and then making it
 
 ## Buy me a coffee? Please don't
 
-Instead spend some money on a charity I care for: [kwf.nl](https://secure.kwf.nl/donation).
+Instead spend some money on a charity I care for: [kwf.nl](https://www.kwf.nl/donatie/donation?icmp=il-86).
```

Comment:
Can you please remove the unneeded `?icmp=il-86` parameter please 😄 ?
Other than that major thanks for keeping the link up-to-date!